### PR TITLE
Lithium + MDMA

### PR DESCRIPTION
**Pull request by utaninja**
Added Lithium + MDMA

### CODE REVIEW

**Review by Electromorphosis — APPROVED** [review]
Sources and status seem ok

**Comment by LunaUrsa** [comment]
I made updates, just FYI:

1. Made the MDMA + Lithium combo, to reflect the Lithuim + MDMA combo (updated the mdma drug)
2. Updated drugs.json with both combos

**Review by LunaUrsa — APPROVED** [review]

File: combos.json [hunk 1]
```diff
@@ -1822,6 +1832,16 @@
       "note": "No unexpected interactions, though likely to increase blood pressure  but not an issue with sensible doses. Moving around on high doses of  this combination may be ill advised due to risk of physical injury.",
       "status": "Low Risk & Synergy"
     },
+    "lithium": {
+      "note": "High risk of serotonin syndrome",
+      "sources": [
+        "https://neuro.psychiatryonline.org/doi/full/10.1176/appi.neuropsych.11080196#:~:text=Lithium%20and%20venlafaxine%20combination%20has,is%20enhances%20the%20serotonin%20toxicity",
```
[the remark this answers is not in the document]
Comment (LunaUrsa) [reply combos.json:1838]: I love the suggestion, thank you, ill start a new discussion about the source field, we have the option to change it if we want!

[the remark this answers is not in the document]
Comment (LunaUrsa) [reply combos.json:1838]: I've made this change on the Main branch, thanks again

**Comment by utaninja** [comment]
> I made updates, just FYI:
> 
> 1. Made the MDMA + Lithium combo, to reflect the Lithuim + MDMA combo (updated the mdma drug)
> 2. Updated drugs.json with both combos

got it! will try to add both next time!

**Pull request merged**